WeartGloves/models/XSenseTestThreeFingers.urdf

```diff
@@ -0,0 +1,2275 @@
+<!--URDF MODEL 48 DoFs with Hands-->
+<!-- +16 DoFs left hand, +16 DoFs right hand -->
```

Comment:
Where did you get this file? Maybe it is worth renaming it with a more appropriate name?